testing/tests/DevExpress.ui.widgets.scheduler/common.tests.js

```diff
@@ -1092,6 +1092,29 @@ QUnit.testStart(function() {
         assert.equal(navigator.option("intervalCount"), 3, "navigator has correct count");
     });
 
+    QUnit.test("view.intervalCount is passed to workspace & header & navigator, currentView is set by view.name", function(assert) {
+        this.createInstance({
+            currentView: "WEEK1",
+            views: [{
+                type: "day",
+                name: "DAY1",
+                intervalCount: 5
+            }, {
+                type: "week",
+                name: "WEEK1",
+                intervalCount: 3
+            }]
+        });
+
+        var workSpaceWeek = this.instance.element().find(".dx-scheduler-work-space").dxSchedulerWorkSpaceWeek("instance"),
```

Comment:
You can get work space instance by using the `this.instance.getWorkSpace()` method.